app.json

```diff
@@ -5,23 +5,40 @@
   "success_url": "/",
   "stack": "container",
   "env": {
-    "SUPPRESS_SELF_INVITE": {
-      "description": "Should anonymous visitors be able to create new organizations from the home page? You will probably want to start the app with this off, and then after creating your own organization, come back here and set it to 1",
```

Comment:
Do we want this removed from all initial applications moving forward?

Reply:
NVM:  This document was alphabetized for user readability moving forward.  The changes look as though SUPPRESS_SELF_INVITE was deleted, but in fact it is still available but on line 206 of the code.